vehicle/mkvi/software/sensing/sensing.h

```diff
@@ -0,0 +1,37 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+/* ATMega 16m1 Pins */
+gpio_t DEBUG_LED = PB0;
+gpio_t WHEEL_SPEED_LEFT = PB2;
+gpio_t WHEEL_SPEED_RIGHT = PD6;
+
+enum { counter_left = 0 };
+enum { counter_right = 0 };
+/*
+ * Timer config
+ */
+void timer1_isr(void);
+
+/*
+ * 1s timer (1kHz)
+ *
+ * F_CPU / prescaler / output_compare_match
+ * 4000000 / 256 / 15625 = 1
```

Comment:
Comment math doesn't match the prescaler you picked

Reply:
also, make sure you change your CTC value in the timer config as well!